Makefile

```diff
@@ -16,7 +16,7 @@ test-harness-ci: build
 	MOCK_ADDRESS=127.0.0.1:8080 NETWORK_SIZE=3 DUSK_BLOCKCHAIN=${PWD}/bin/dusk DUSK_UTILS=${PWD}/bin/utils DUSK_SEEDER=${PWD}/bin/voucher DUSK_WALLET_PASS="password" make test-harness
 test-harness-alive: stop build
 	MOCK_ADDRESS=127.0.0.1:9191 NETWORK_SIZE=3 DUSK_BLOCKCHAIN=${PWD}/bin/dusk DUSK_UTILS=${PWD}/bin/utils DUSK_SEEDER=${PWD}/bin/voucher DUSK_WALLET_PASS="password" \
-	go test -v --count=1 --test.timeout=0 ./harness/tests/ -args -enable -keepalive
+	go test -v --count=1 --test.timeout=0 ./harness/tests/ -args -enable 
```

Comment:
Maybe this was a consequence of merging with `rusk-ipc`, if not, why is the `-keepalive` flag removed?

Reply:
it was a mere oversight. Reintroduced